migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc

```diff
@@ -9,12 +9,18 @@ toc::[]
 You can migrate your applications by using the {mtc-full} ({mtc-short}) web console or from the command line.
 
 include::modules/migration-prerequisites.adoc[leveloffset=+1]
+
+[discrete]
+=== Additional resources
```

Comment:
Can you add an ID here?  Please see my next comment.

Some day, we'll be able to put this link a little closer to that second bullet.  Our poor customers.